config.rb

```diff
@@ -163,4 +163,12 @@ def self.e2e_test?
   # Load Balancer
   optional :load_balancer_service_project_id, string
   optional :load_balancer_service_hostname, string
+
+  # Let's Encrypt
+  # The following are optional because they are only needed in production.
+  # They are not needed in development or test.
+  optional :acme_email, string
+  override :acme_directory, "https://acme.zerossl.com/v2/DV90", string
```

Comment:
The comment mentions 'Let's Encrypt', but the directory is 'ZeroSSL'. Aren't they two different providers?

Reply:
really it's anything ACME, but only total nerds know about ACME, which was developed by ISRG for the Let's Encrypt project.

Reply:
removed the `Let's Encrypt` all together, variable names are clear enough.